src/main/java/org/spongepowered/api/service/economy/Account.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.service.economy;
+
+import java.math.BigDecimal;
+import java.util.Map;
+import java.util.Set;
+
+import org.spongepowered.api.service.context.Context;
+import org.spongepowered.api.service.context.Contextual;
+
+/**
+ * TODO: Add overview doc
+ *
+ */
+public interface Account extends Contextual {
```

Comment:
Does it make sense that an Account is a Contextual object?

``Subject`` is ``Contextual`` because it is located in a world/dimension that can be fetched by a ContextCalculator.

An Account exists in all dimensions, there is no way to get the world/dimension/etc. of an Account. That means the getActiveContexts() method of the Contextual interface doesn't make sense for an Account.